crates/p2p_proto/proto/sync.proto

```diff
@@ -80,3 +72,13 @@ message Status {
   starknet.common.FieldElement hash = 2;
   starknet.common.FieldElement chain_id = 3;
 }
+
+message GetContractClasses {
+  repeated starknet.common.FieldElement class_hashes = 1;
+  uint64 count = 2;
```

Comment:
What exactly does `count` mean here? If it's the number of class hashes requested then I think it's redundant.